.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @aherrmann @mboes
```

Comment:
This file doesn't exist in `rules_haskell`, but was listed [here](https://github.com/tweag/guides/blob/master/project-management/Github.md#project-leads).